recipes/tweeny/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
+
+
+required_conan_version = ">=2.0.9"
+
+class TweenyConan(ConanFile):
+    name = "tweeny"
+    description = "Tweeny is an inbetweening library designed for the creation of complex animations for games and other beautiful interactive software"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mobius3/tweeny"
+    topics = ("animation", "tweening", "easing-functions")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        check_min_cppstd(self, 11)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5"  # CMake 4 support
```

Comment:
this is already fixed in the most recent revision, may want to add a comment so that if / when a new version is added, this is limited to this older version? (or a conditional, even)